scripts/ci-entrypoint.sh

```diff
@@ -141,6 +141,16 @@ wait_for_nodes() {
     "${KUBECTL}" get nodes -owide
 }
 
+copy_secret() {
+    unset KUBECONFIG
+    "${KUBECTL}" get secret "${CLUSTER_NAME}-control-plane-azure-json" -o jsonpath='{.data.control-plane-azure\.json}' | base64 --decode > azure_json
+    
+    export KUBECONFIG="${KUBECONFIG:-${PWD}/kubeconfig}"
+    kubectl create secret generic "${CONFIG_SECRET_NAME}" -n kube-system \
```

Comment:
I'm not sure I understand why we need to copy the secret since there is already an existing secret with this information, is the issue that CCM expects the key to be "cloud-config"? Is that configurable?

Reply:
Thanks for the review. Because the existing secret is in the mgmt cluster, CCM expects the secret from the worker cluster. So we need to copy secret from mgmt cluster to worker cluster.

Reply:
> I'm not sure I understand why we need to copy the secret since there is already an existing secret with this information, is the issue that CCM expects the key to be "cloud-config"? Is that configurable?

Thanks for the review. Because the existing secret is in the mgmt cluster, CCM expects the secret from the worker cluster. So we need to copy secret from mgmt cluster to worker cluster.